.infrastructure/docker/run.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
+echo "starting $1"
+# we do the weirdest hack of them all. if the "PORT" variable is exposed
+# we _are_ the web process. so start web and otherwise, start the worker.
+# see  https://github.com/dokku/dokku/blob/v0.4.14/dokku#L143-L159
+cd /app
+if [[ -n "$PORT" ]]; then
+  echo "Running migrations"
+  python manager.py db upgrade heads
+  echo "Starting Web Service"
+  start web
+else
+  echo "Starting Worker Service"
+  start worker
```

Comment:
the [celery] worker queue/processes. a worker queue (currently not much in use) to run background processes that otherwise take too long or need to run independent from requests coming in (like sending/receiving email, fetching images and such).